.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,10 @@
+PLease provide the following details along with the issue - Thank you for the contribution.
+
+* `node` version is:
```

Comment:
To make this more beginner friendly, maybe provide the commands to get the version (or where to find it).